kszeto20/BELPdev
# Fix showing listing index page

## Changes
* replaced any plural "listings" with "listing" in class names and directories to match how catalog was donee
* deleted old migrations and created new ones with "listing" (non plural)
## How to test
* pull branch locally
    * `git fetch` (downloads all remote branches)
    * `git checkout fix-showing-bathroom-index-page`
* run migrations (`./manage.py migrate`)
* navigating to localhost:8000 should redirect to localhost:8000/listing
